tests/executequery_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !emulator
+// +build !emulator
+
+package tests
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/googleapis/cloud-bigtable-clients-test/testproxypb"
+	"github.com/stretchr/testify/assert"
+	btpb "google.golang.org/genproto/googleapis/bigtable/v2"
+	"google.golang.org/protobuf/testing/protocmp"
+)
+
+func TestExecuteQuery_EmptyResponse(t *testing.T) {
```

Comment:
nit, can we add some comments on which step is doing what? similar to other test files, so it's easy to read.